CONTRIBUTING.rst

```diff
@@ -28,7 +28,7 @@ Co-develop Zenodo
 -----------------
 Are you missing a feature in Zenodo? Why not help us built the feature. We are
 happy to receive both tiny and large contributions to Zenodo. Have a look at
-our `Development Process <https://github.com/zenodo/zenodo/blob/master/docs/development/process.rst>`_ and `Project Life Cycle <https://github.com/zenodo/zenodo/blob/master/docs/development/projectlifecycle.rst>`_ and don't hesitate to get in
+our :ref:`development_process_` and :ref:`project_life_cycle`and don't hesitate to get in 
```

Comment:
the labels have to match, e.g. ``_project_life_cycle`` and ``_development_process``. Please compile it locally and verify that the docs indeed build and the links works (see ``./run-tests.sh`` which builds it like this: ``sphinx-build -qnNW docs docs/_build/html``)

Reply:
OK, will try. I presume the "Coverage decreased" can be ignored.

Reply:
Yes, the coverage dropped because the tests failed. Also make sure to rebase to master branch.

Reply:
That's the one I branched off from this morning. At least one correct intuition ;-) Thanks for helping me get started with reST!

Reply:
The master branch has changed since